### ISSUE TITLE
Issue with Excessive Progress Bar Updates in YOLOv8 Training

### ISSUE THREAD

**Issue by superdoger** [ultralytics/ultralytics#17853]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I am currently training a model using YOLOv8 and have encountered an issue where an unnecessary progress bar is being printed to the console.，0%|          | 0/360 [00:00<?, ?it/s].
![屏幕截图 2024-11-28 200402](https://github.com/user-attachments/assets/1c9759ad-d9ca-498c-a517-b220f438c42c)

**Comment by UltralyticsAssistant** [comment]
👋 Hello @superdoger, thank you for your interest in Ultralytics 🚀! We recommend a visit to the [Docs](https://docs.ultralytics.com) for new users where you can find many [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples and where many of the most common questions may already be answered.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) to help us debug it. This includes:
1. The exact training command you are using
2. Details about your environment (e.g., OS, Python version, hardware)
3. Any relevant custom configurations/settings

If this is a custom training ❓ Question, please provide as much information as possible, including dataset image examples and training logs, and verify you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com). Or dive into threads on our [Subreddit](https://reddit.com/r/Ultralytics) to share knowledge with the community.

## Upgrade

Before continuing, please make sure you are using the latest `ultralytics` release and all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) are satisfied in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/):

```bash
pip install -U ultralytics
```

## Environments

YOLO may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

This is an automated response, but rest assured that an Ultralytics engineer will take a closer look at your issue as soon as possible 🔍. Thank you for your patience! 😊

**Comment by Y-T-G** [comment]
What's the training command?

Can you provide the output after running this command in terminal: `yolo checks`?

**Comment by superdoger** [comment]
The training command is as follows.

from ultralytics import YOLO
import os
# 训练
model =YOLO(r'D:\Downloads\ultralytics-8.3.9\ultralytics\cfg\models\v8\yolov8x-AFPN.yaml')
model.train(data=r'D:\Downloads\ultralytics-8.3.9\datasets\GH1028\GH1028.yaml',workers=0,epochs=400,batch = 0.9,patience = 50,amp = False)

The result of executing the YOLO checks is as follows.

Ultralytics 8.3.9 🚀 Python-3.12.7 torch-2.4.1+cu124 CUDA:0 (NVIDIA GeForce RTX 4060 Ti, 16380MiB)
Setup complete ✅ (12 CPUs, 31.6 GB RAM, 129.1/753.9 GB disk)

OS                  Windows-11-10.0.22631-SP0
Environment         Windows
Python              3.12.7
Install             pip
RAM                 31.58 GB
Disk                129.1/753.9 GB
CPU                 AMD Ryzen 5 7500F 6-Core Processor
CPU count           12
GPU                 NVIDIA GeForce RTX 4060 Ti, 16380MiB
GPU count           1
CUDA                12.4

numpy               ✅ 1.26.3>=1.23.0
matplotlib          ✅ 3.9.2>=3.3.0
opencv-python       ✅ 4.10.0.84>=4.6.0
pillow              ✅ 10.2.0>=7.1.2
pyyaml              ✅ 6.0.2>=5.3.1
requests            ✅ 2.32.3>=2.23.0
scipy               ✅ 1.14.1>=1.4.1
torch               ✅ 2.4.1+cu124>=1.8.0
torchvision         ✅ 0.19.1+cu124>=0.9.0
tqdm                ✅ 4.66.5>=4.64.0
psutil              ✅ 6.0.0
py-cpuinfo          ✅ 9.0.0
pandas              ✅ 2.2.3>=1.1.4
seaborn             ✅ 0.13.2>=0.11.0
ultralytics-thop    ✅ 2.0.9>=2.0.0
torch               ✅ 2.4.1+cu124!=2.4.0,>=1.8.0; sys_platform == "win32"

**Comment by Y-T-G** [comment]
Does it occur if you zoom out the terminal a bit?

**Comment by superdoger** [comment]
Yes, it occurs regardless of how I scale the terminal.

**Comment by Y-T-G** [comment]
You can try upgrading to the latest version.
